crawler/features_crawler.py

```diff
@@ -541,10 +562,13 @@ def crawl_processes(self):
 
     def _crawl_processes(self):
 
-        created_since = 0
+        created_since = -1
```

Comment:
I don't think `created_since` is supported in most of these functions. What's the point of making it `-1`?